### DISCUSSION TITLE
Error: Unable to load plugin instance. Ensure plugin is publicly accessible

### DISCUSSION THREAD

**Discussion by ghost** [capawesome-team/capacitor-firebase#693] [Q&A]
Hi, I'm testing the @capacitor-firebase/authentication package with version = 6.1.0. On web is working perfectly but when I tried to test it on Android I am receiving the following error: 

![image](https://github.com/user-attachments/assets/2ef51dc3-2b16-4f82-a201-d7d9e619bed2)

I have followed the package install instructions, executed npx cap sync and did the resync gradle packages. Can someone help me with this?

This is the only code I am testing: 

`initializeApp({
        projectId:"...",
        appId:"...",
        storageBucket:"...",
        apiKey:"...",
        authDomain:"...",
        messagingSenderId:"..."
    })
    
    const result = await FirebaseAuthentication.signInWithGoogle()
    
    console.log(result)`

**Comment by secret-agent-B** [comment]
I am experiencing the same issue. Any luck?

**Reply by secret-agent-B** [reply]
I was able to resolve the issue by setting up an Android app under the Project Settings. It's very important that you follow the directions for the google-settings.json file. 

![image](https://github.com/user-attachments/assets/45944877-907c-41ea-a640-f7673e5dfb1e)
